src/wagtail_personalisation/migrations/0021_segment_status_as_boolean.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2017-08-10 13:48
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtail_personalisation', '0020_rules_delete_relatedqueryname'),
+    ]
+
+    operations = [
```

Comment:
Status data will be lost without a multi-step migration which:

1. create the new field
1. populate the new field from the old one
1. delete the old field